lib/client/browser.cjs

```diff
@@ -169,8 +169,6 @@ function getTestPageName(qunitHtmlFile) {
 			config.urlParameters.forEach(function(urlParameter) {
 				url.searchParams.append(urlParameter.key, urlParameter.value);
 			});
-			// Sort params for consistency between browsers (probably caused by polyfill)
```

Comment:
I could not find out what consistency this comment relates to. It might became obsolete after switching the polyfills via https://github.com/SAP/karma-ui5/pull/120